Missing inttypes.h with clang

```
  CC       ldap/servers/snmp/agent-main.o
/home/william/development/389ds/ds/ldap/servers/snmp/main.c:74:52: error: expected ')'
            printf("%s pid is %d - sleeping for %" PRId64 "\n", argv[0], getpid(), secs);
                                                   ^
/home/william/development/389ds/ds/ldap/servers/snmp/main.c:74:19: note: to match this '('
            printf("%s pid is %d - sleeping for %" PRId64 "\n", argv[0], getpid(), secs);

```

Due to missing intypes.h in main.c.

--- a/ldap/servers/slapd/main.c
+++ b/ldap/servers/slapd/main.c
@@ -2838,7 +2838,7 @@ slapd_debug_level_log(int level)
     }
 
     /* second pass: construct the debug level string */
-    p = msg = slapi_ch_malloc(len);
+    p = msg = slapi_ch_calloc(1, len);
     count = 0;
     for (i = 0; NULL != slapd_debug_level_map[i].dle_string; ++i) {
         if (!slapd_debug_level_map[i].dle_hide &&

--- a/ldap/servers/snmp/main.c
+++ b/ldap/servers/snmp/main.c
@@ -15,6 +15,7 @@
 #include <string.h>
 #include <stdio.h>
 #include <unistd.h>
+#include <inttypes.h>
 #include <signal.h>
 #include <sys/stat.h>
 #include "ldap-agent.h"

